python_ta/checkers/forbidden_import_checker.py

```diff
@@ -1,10 +1,18 @@
+"""Checker or use of forbidden imports.
+"""
+
+import inspect
```

Comment:
This module is not used, so should be removed (this is likely leftover from the merge conflict).